src/ui-tray-icon.c

```diff
@@ -154,18 +174,26 @@ pixbuf_array_new(int size)
 	system_theme = prefs_get_boolean("SystemTheme", FALSE);
 
 	if (system_theme) {
-		pixbufs[VOLUME_MUTED] = pixbuf_new_from_stock("audio-volume-muted", size);
-		pixbufs[VOLUME_OFF] = pixbuf_new_from_stock("audio-volume-off", size);
-		pixbufs[VOLUME_LOW] = pixbuf_new_from_stock("audio-volume-low", size);
-		pixbufs[VOLUME_MEDIUM] = pixbuf_new_from_stock("audio-volume-medium", size);
-		pixbufs[VOLUME_HIGH] = pixbuf_new_from_stock("audio-volume-high", size);
-		/* 'audio-volume-off' is not available in every icon set.
-		 * Check freedesktop standard for more info:
-		 *   http://standards.freedesktop.org/icon-naming-spec/
-		 *   icon-naming-spec-latest.html
-		 */
-		if (pixbufs[VOLUME_OFF] == NULL)
-			pixbufs[VOLUME_OFF] = pixbuf_new_from_stock("audio-volume-low", size);
+		pixbufs[VOLUME_MUTED] = pixbuf_new_from_stock(
+		(const gchar*[]) {
+			"audio-volume-muted-panel", "audio-volume-muted", NULL
+		}, size);
+		pixbufs[VOLUME_OFF] = pixbuf_new_from_stock(
+		(const gchar*[]) {
+			"audio-volume-off-panel", "audio-volume-off", "audio-volume-low-zero-panel", "audio-volume-low-panel", "audio-volume-low", NULL
```

Comment:
I'd add `audio-volume-low-zero` after `audio-volume-low-zero-panel` for completeness